src/types/libdef/npm/idb_v5.x.x.js

```diff
@@ -0,0 +1,643 @@
+// Derived from TS typings at https://github.com/jakearchibald/idb/blob/c8bab898f01bacab96097f87c1e42a0c19e01425/lib/idb.d.ts
```

Comment:
This has been contributed to flow-typed (https://github.com/flow-typed/flow-typed/pull/3857)